package.json

```diff
@@ -13,7 +13,8 @@
     "speculate": "speculate",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test:e2e": "npx cypress run",
```

Comment:
Both this and the next line fail after `npm run dev` because they expect `npm run storybook`. L17 doesn't need the baseURL passed like this, cypress project can be used https://docs.cypress.io/guides/guides/command-line.html#Run-tests-specifying-a-project with a separate config for the storybook folder than the main repo. Example of setup is here: https://github.com/cypress-io/cypress-test-nested-projects

I think this alternative implementation would be a lot tidier and be the normal way to split projects using cypress.